src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -54,6 +66,14 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
     const classes = useStyles();
     const intl = useIntl();
     const timer = React.useRef();
+    const [isConfirmationPopupOpen, setOpenConfirmationPopup] = useState(false);
+    const [currentScript, setCurrentScript] = useState(null);
+    const currentCriteriaBasedFilter = useRef(null);
```

Comment:
this should be a state... why it's a useRef ?

Reply:
We tried to change it to a state, like we did with https://github.com/gridsuite/gridexplore-app/pull/217 , but couldn't, it produces bugs we did not succeed in fixing in time.